recipes/base_install.rb

```diff
@@ -64,6 +64,9 @@
   version '1.15.40'
 end
 
+# Install boto3
+python_package 'boto3'
```

Comment:
Is this really needed? It is not already in some dependency chain?

Reply:
yes, boto2 is installed as part of `awscli`. The base AMI does not have boto3 installed.